packages/create-fuels/package.json

```diff
@@ -22,12 +22,10 @@
     "chalk": "4",
     "commander": "^9.4.1",
     "fuels": "workspace:*",
-    "prompts": "^2.4.2",
-    "tar": "^6.2.0"
```

Comment:
Removed this unused dependency that I came across while scanning for the usage of the `tar` command